README.md

```diff
@@ -1,110 +1,29 @@
-# ingress-controller-conformance
+# ingress-conformance-tests
```

Comment:
I'd leave the title as-is, to match the repo name.

Reply:
fixed